### PR TITLE
Fix `dunstctl rule` error when not providing rule state

### PR DESCRIPTION
**Pull request by SeerLite**
Previously showed "line 130: 3: unbound variable" instead of the pretty error message.

### CODE REVIEW

File: dunstctl [hunk 1]
```diff
@@ -126,12 +126,16 @@ case "${1:-}" in
...
 		[ "${2:-}" ] \
 			|| die "No rule name parameter specified. Please give the rule name"
-		state=nope
-		[ "${3}" = "disable" ] && state=0
-		[ "${3}" = "enable" ]  && state=1
-		[ "${3}" = "toggle" ]  && state=2
-		[ "${state}" = "nope" ] \
-			&& die "No valid rule state parameter specified. Please give either 'enable', 'disable' or 'toggle'"
+		case "${3:-}" in
+			("disable")
```
[the remark this answers is not in the document]
Comment (SeerLite) [reply dunstctl:130]: You're right! However I was basing this style on this same file [here](https://github.com/dunst-project/dunst/blob/70d9c60c0a2cab50899251e57940265db68bc1b5/dunstctl#L147-L152)

[the remark this answers is not in the document]
Comment (SeerLite) [reply dunstctl]: Updated to use the same style as the main `case`. I didn't touch the other unrelated `case` though.

**Comment by bynect** [comment]
So the problem was the reference to `"${3}"`?

**Comment by SeerLite** [comment]
Yes, the first commit just changes `"$3"` to `"${3:-}"` and second one replaces with the `if` with a `case`

**Comment by bynect** [comment]
Thanks 👍🏻

**Pull request merged**